test/GoldenToken.ts

```diff
@@ -29,15 +36,68 @@ describe('GoldenToken - ERC20 token', function () {
     it('Should have correct token total supply', async function () {
       expect(await GoldenToken.totalSupply()).to.equal(INITIAL_SUPPLY);
     });
+
+    it('Should set owner with transfer grant', async function () {
+      expect(await GoldenToken.hasGrantsToTransfer(owner.address)).to.be.true;
+
+      for (const user of users) {
+        expect(await GoldenToken.hasGrantsToTransfer(user.address)).to.be.false;
+      }
+    });
   });
 
-  describe('Transactions', function () {
-    it('Should transfer tokens from owner', async function () {
-      const balance = await GoldenToken.balanceOf(users[0].address);
-      await owner.GoldenToken.transfer(users[0].address, 50);
-      expect(await GoldenToken.balanceOf(users[0].address)).to.equal(
-        balance.add(50)
-      );
+  describe(`Transfers`, function () {
```

Comment:
nit: shouldn't these tests be in TransferRole.ts?

Reply:
I think it shouldn't, because functionality of ERC20 has been modified not of the TransferRole.

Reply:
I think it shouldn't because the functionality of ERC20 has been modified not of the TransferRole.